src/EditorFeatures/Test2/FindReferences/FindReferencesTests.NamedTypeSymbols.vb

```diff
@@ -2156,5 +2156,35 @@ public class D { }
 
             Test(input)
         End Sub
+
+        <Fact, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        <WorkItem(1174256)>
+        Public Sub TestFarWithInternalsVisibleToNull()
+            Dim input =
+<Workspace>
+    <Project Language="C#" AssemblyName="ClassLibrary1" CommonReferences="true">
+        <Document><![CDATA[
+        [assembly: System.Runtime.CompilerServices.InternalsVisibleTo(null)]
+        internal class A
+        {
+        }]]>
+        </Document>
+    </Project>
+
+    <Project Language="C#" AssemblyName="ClassLibrary2" CommonReferences="true">
+        <ProjectReference>ClassLibrary1</ProjectReference>
+        <Document><![CDATA[
+        public class B
+        {
+            [$$A] _a;
```

Comment:
Did you mean ```[|$$A|]``` to indicate that the ```A``` should be found? As it is I think this will result in a C# file with literally ```[A]``` in it.